crates/broker/src/lib.rs

```diff
@@ -172,7 +172,7 @@ fn format_order_id(
 /// Order request from the network.
 ///
 /// This will turn into an [`Order`] once it is locked or skipped.
-#[derive(Serialize, Deserialize, Debug)]
+#[derive(Serialize, Deserialize, Debug, Clone)]
```

Comment:
```suggestion
#[derive(Serialize, Deserialize, Debug)]
```

In relation to previous comment, would be nice to avoid this impl to avoid race conditions where we have duplicates of the same order in different states accidentally

Reply:
Thats fair. We should only be logging order id + expiry times (via the Display impl), so should be safe, but generally agree